tests/deepsparse/pipelines/test_clip.py

```diff
@@ -30,24 +32,48 @@
 from tests.utils import mock_engine
 
 
+def custom_process_inputs(self, inputs):
+    if not isinstance(inputs.text, list):
+        inputs.text = [inputs.text]
+    if not isinstance(inputs.text[0], str):
+        return inputs.text
+    tokens = [np.array(t).astype(np.int32) for t in self.tokenizer(inputs.text)]
+    tokens = np.stack(tokens, axis=0)
+    tokens_lengths = np.array(tokens.shape[0] * [tokens.shape[1] - 1])
+    return [tokens, tokens_lengths]
+
+
+# This overrides the process_inputs function globally for all CLIPTextPipeline classes.
+# This is needed for CLIP-ViT-B-32-256x256-DataComp-s34B-b86K as it has a second input
+# that specifies how many tokens are present.
+CLIPTextPipeline.process_inputs = custom_process_inputs
+
+
 @pytest.fixture
-def visual_input():
+def model_folder():
+    from huggingface_hub import snapshot_download
+
+    model_id = "neuralmagic/CLIP-ViT-B-32-256x256-DataComp-s34B-b86K-quant-ds"
```

Comment:
Could we add a quick comment/note indicating that this model is not from OpenCLIP and only use for zero-shot classification?